src/chainparams.cpp

```diff
@@ -100,6 +100,7 @@ class CMainParams : public CChainParams {
         consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000003f94d1ad391682fe038bf5");
 
         consensus.HardforkTime = std::numeric_limits<int64_t>::max();
+        consensus.nPowChangeTargetShift = 20;
```

Comment:
Does this make it scale up by 2^20 whenever the previous period's blocks were too fast? I don't totally grok this constant.

Reply:
It makes it 2^20 easier whenever the PoW algo changes.